.github/workflows/ci.yaml

```diff
@@ -47,3 +47,33 @@ jobs:
         if: success() || failure() # still run black if above checks fails
         run: |
           black --check --verbose .
+  regression:
+    runs-on: ubuntu-22.04
+    permissions:
+      id-token: write # This is required for requesting the JWT
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .
+      # See https://github.com/aws-actions/configure-aws-credentials for configuring
+      # the aws side of this - the below action is just a light wrapper
+      - name: Configure AWS Credentials
+        uses: mcblair/configure-aws-profile-action@v0.1.1
+        with:
+          role-arn: arn:aws:iam::316333106222:role/cumulus-library-ci
```

Comment:
Should this be a repo secret? I don't know how sensitive IAM numbers are.

Wait, speaking of - this is actually hitting athena, right? Is this cumulus-library-ci role wide open? (i.e. if I fork this repo and run CI, I'll be able to hit the same files) I don't see obvious repo secrets.

Reply:
From aws: _While account IDs, like any identifying information, should be used and shared carefully, they are not considered secret, sensitive, or confidential information._ - that said, i could make this one a secret.

The real security comes in on the AWS side - over there, we have an identity provider to allow AWS traffic from github's official integration, and only when it comes from our github org and the cumulus-library repo, and it only gives them access to the sythetic regression S3 locations.

Reply:
Ah cool, that makes sense.